package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "mqtt",
   "description": "A library for the MQTT protocol",
-  "version": "5.0.4",
```

Comment:
Please don't bump the version. I have an automated process that does this on release

Reply:
"umbumped" the version